app/services/mobilize.py

```diff
@@ -46,36 +44,49 @@ def database_error(context: str) -> JSONResponse:
     return JSONResponse(status_code=502, content={"detail": message})
 
 
-@mobilize.get("/uploadcsv", response_class=HTMLResponse)
+@mobilize.get("/uploadcsv", response_class=FileResponse)
 async def upload_csv(request: Request):
-    return templates.TemplateResponse("upload_csv.html", {"request": request})
+    return FileResponse("docs/upload_csv.html")
```

Comment:
Just remove this method entirely.  No point in having an API when people can just directly load "/docs/upload_csv.html" in their browser.  (And then you can remove the FileResponse import.)